src/CommunityToolkit.Maui.Core/Essentials/SpeechToText/SpeechToTextImplementation.windows.cs

```diff
@@ -119,7 +119,10 @@ async Task StopRecording()
 	{
 		try
 		{
-			await speechRecognizer?.ContinuousRecognitionSession.StopAsync();
+			if (speechRecognizer is not null)
+			{
+				await speechRecognizer.ContinuousRecognitionSession.StopAsync();
+			}
```

Comment:
the before was `await speechRecognizer?.ContinuousRecognitionSession.StooAsync();`
which will be compiled as what you wrote. See the method M [here](https://sharplab.io/#v2:CYLg1APgAgTAjAWAFBQAwAIpwKwG5nJQDMmM6AwugN7Lp2YlQBs6AsgBQCU1t9fAygHsAtgFNyAGwCGAZxkB+dAGMAvHQB2AVwkT8SPn14HMAdmXyAdABFBAQRkBPdUq56+AX2SekBFGSFikrIyyDT69MSYLDb2Ti6cRjzhxlBmzNai0g7scKionG703u5AA)

Reply:
because you created different code. add await before c: 
![image](https://github.com/CommunityToolkit/Maui/assets/33021114/321f1706-67c3-4993-958d-cc3c6dcca5bc)

Reply:
No, that's because I'm returning the value, to make the lowering more readable, if I do `Task?` as a return the warning goes way, as you can see [here](https://sharplab.io/#v2:CYLg1APgAgTAjAWAFBQAwAIpwKwG5nJQDMmM6AwugN7Lp2YlQBsA/OgLIAUAlNbfQIDKAewC2AU3IAbAIYBnOWwDGAXjoA7AK5Sp+JAIH8DmAOzolLAHQARYQEE5AT3VKeegQF9kXpARRkRCWl5OWQafXpiTCZ0WwdnV24jPgjjKDNmG3FZR044VFRud3ofDyA==)

Reply:
hm... Interesting